### DISCUSSION TITLE
What is the release cadence for fwupd?

### DISCUSSION THREAD

**Discussion by jkolako-solidigm** [fwupd/fwupd#5337] [Q&A] [answered]
I see that there is a new version roughly once a month. Is there any release plan that is being followed? When can contributors expect their changes to be in official release?

**Comment by hughsie** [comment, marked as answer]
There's no formal release plan [maybe we should have a formal plan] but you're correct that it's normally about once per month. Tagging releases isn't super expensive, and I can do an "early" one if that would help you. What date do you need the new release for in an ideal world?

**Reply by jkolako-solidigm** [reply]
Thank you for the reply. Would it be possible to have new release next week?

**Reply by hughsie** [reply]
Next week no, I'm afraid I'm off for Christmas until 3rd January. I can do a release the first week of January if that helps.

**Reply by jkolako-solidigm** [reply]
That would be great, thank you. Merry Christmas and Happy New Year.
